src/applications/caregivers/components/ConfirmationPage/ConfirmationPrintView.jsx

```diff
@@ -57,7 +58,7 @@ const ConfirmationPrintView = ({ name, timestamp }) => {
 
 ConfirmationPrintView.propTypes = {
   name: PropTypes.object,
-  timestamp: PropTypes.string,
+  timestamp: PropTypes.number,
```

Comment:
Removing console warning since the prop is just a unix timestamp and not a string.